src/main/resources/migrations.xml

```diff
@@ -5848,4 +5848,44 @@
 		</sql>
 	</changeSet>
 
+	<changeSet id="add-entityAvatars-table" author="fdelcastillo">
+		<createTable tableName="entityAvatars">
+			<!-- The next two columns combined act as a generic foreign key reference to a (table name, uuid)-tuple -->
+			<column name="relatedObjectType" type="varchar(255)">
+				<constraints nullable="false" />
+			</column>
+			<column name="relatedObjectUuid" type="${uuid_type}">
+				<constraints nullable="false" />
+			</column>
+			<column name="attachmentUuid" type="${uuid_type}">
+				<constraints nullable="false" />
+			</column>
+			<column name="applyCrop" type="boolean">
+				<constraints nullable="false" />
+			</column>
+			<column name="cropLeft" type="number">
```

Comment:
This seems like a rather big and costly change to be done at this stage for not a lot of gain, I would rather move on and focus on higher priority things. Very low priority in my opinion.

Reply:
The gain will be when we change the people avatars to use this same mechanism. Since we already know we are going to do this next, it would save some refactoring later.